tools/bridge/bridge/confirmation_sender.py

```diff
@@ -0,0 +1,105 @@
+import logging
+from typing import Any, Dict
+
+import gevent
+from gevent.queue import Queue
+
+from web3.contract import Contract
+from eth_keys.datatypes import PrivateKey
+from eth_utils import keccak, int_to_big_endian
+
+from bridge.constants import HOME_CHAIN_STEP_INTERVAL
+
+
+class ConfirmationSender:
+    """Sends confirmTransfer transactions to the home bridge contract."""
+
+    def __init__(
+        self,
+        transfer_queue: Queue,
+        home_bridge_contract: Contract,
+        private_key: bytes,
+        gas_price: int,
+        max_reorg_depth: int,
+    ):
+        self.transfer_queue = transfer_queue
+        self.home_bridge_contract = home_bridge_contract
+        self.private_key = private_key
```

Comment:
Question: So apparently we decided for one version to handle transaction signing. Did we fixed this somewhere? Please add this design decision to the architecture wiki page if not. Else I would be happy be pointed to. :upside_down_face:

Reply:
I don't know if it's written down somewhere. But do we need to document every single decision? It doesn't really have any long lasting effects and it sounds like unnecessary overhead.

Reply:
If you think so. For me as a user (bridge validator in this case), it is quite important. Since the set of bridge validators equals the sealing entities, I would be happy to know if I just can't use my already running validator node with unlocked account. Now we need to write a guide how to handle the valdiator key with the bridge client. So for me this is not a small decision, because it cost additional work and affects the end-user.